build/tutorials/platform/port-your-ethereum-dapp.md

```diff
@@ -0,0 +1,145 @@
+
+C-Chain runs a fork of [go-ethereum](https://geth.ethereum.org/docs/rpc/server)\) called [coreth](https://github.com/ava-labs/coreth) that has the networking and consensus portions replaced with Avalanche equivalents. What's left is the Solidity VM, which runs the smart contracts and manages data structures and blocks on the chain. As a result you get a blockchain that can run all the Solidity smart contracts from Ethereum, but with much greater transaction bandwidth and instant finality that [Avalanche's revolutionary consensus](../../../learn/platform-overview/avalanche-consensus.md) enables.
+
+Coreth is loaded as a plugin into [AvalancheGo](https://github.com/ava-labs/avalancheg), the client node application used to run Avalanche network.
+
+As far as your dapp is concerned, it will be running the same as on Ethereum, just quicker and cheaper. Let's find out how.
+
+## Accessing Avalanche C-Chain
+
+C-Chain exposes the [same API](../../avalanchego-apis/contract-chain-c-chain-api.md) as go-ethereum, so you can use all the familiar APIs that are available on Ethereum for interaction with the platform.
+
+There are multiple ways of working with the C-Chain.
+
+### Through MetaMask
+
+You can access C-Chain through MetaMask, by defining a custom network. Go to MetaMask, log in, click the network dropdown, and select 'Custom RPC'. Data for Avalanche is as follows.  
+
+#### **Avalanche Mainnet Settings:**
+
+* **Network Name**: Avalanche Mainnet C-Chain
+* **New RPC URL**: [https://api.avax.network/ext/bc/C/rpc](https://api.avax.network/ext/bc/C/rpc)
+* **ChainID**: `0xa86a`
+* **Symbol**: `AVAX`
+* **Explorer**: [https://cchain.explorer.avax.network/](https://cchain.explorer.avax.network/)
+
+#### **FUJI Testnet Settings:**
+
+* **Network Name**: Avalanche FUJI C-Chain
+* **New RPC URL**: [https://api.avax-test.network/ext/bc/C/rpc](https://api.avax-test.network/ext/bc/C/rpc)
+* **ChainID**: `0xa869`
+* **Symbol**: `AVAX`
+* **Explorer**: [https://cchain.explorer.avax-test.network](https://cchain.explorer.avax-test.network/)
+
+In your application's web interface, you can [add Avalanche programmatically](../smart-contracts/add-avalanche-to-metamask-programmatically.md) so your users don't have to enter the network data manually. To see the adding custom network flow in action, check out [Pangolin DEX](https://app.pangolin.exchange/).
+
+### Using the public API nodes
+
+Instead of proxying network operations through MetaMask, you can use the public API, which consists of a number of load-balanced AvalancheGo nodes behind a load balancer.
+
+The C-Chain API endpoint is [https://api.avax.network/ext/bc/C/rpc](https://api.avax.network/ext/bc/C/rpc) for the main net and [https://api.avax-test.network/ext/bc/C/rpc](https://api.avax-test.network/ext/bc/C/rpc) for the testnet.
+
+### Running your own node
+
+If you don't want for your dapp to depend on a centralized service you don't control, you can run your own node and access the network that way.
```

Comment:
I'd also add that running your own node is an easy way to get around rate-limiting